.github/workflows/ci-core.yml

```diff
@@ -287,7 +287,14 @@ jobs:
         if: ${{ needs.filter.outputs.changes == 'true' }}
         uses: actions/download-artifact@c850b930e6ba138125429b7e5c93fc707a7f8427 # v4.1.4
         with:
+          name: go_core_tests_logs
           path: ./artifacts
+      - name: Delete go_core_tests_logs/coverage.txt
+        if: ${{ needs.filter.outputs.changes == 'true' }}
```

Comment:
```suggestion
        if: needs.filter.outputs.changes == 'true'
```

Total nit but :)

Reply:
All the other lines are wrapped with the brackets, so i’m just going to leave it for conformity purposes.